data.lua

```diff
@@ -154,6 +154,19 @@ data{
 	}
 }
 
+data{
+	file = "sp_municipalities.shp",
+	summary = "Sao Paulo municipalities.",
```

Comment:
Please add an example with this data to the package in order to make it easier to verify the functionality.